client.py

```diff
@@ -96,7 +96,7 @@ async def handle_add_game_request(message, entry):
         elif status ==  GameAddStatus.FREE_GAME:
             reply = 'This game is free'
         else:
-            reply = (f'**{entry["name"].capitalize()}** was successfully added to our tracking list.\nsteam://openurl/https://store.steampowered.com/app/{entry["appid"]}\n')
+            reply = (f'**{entry["name"].title()}** was successfully added to our tracking list.\nsteam://openurl/https://store.steampowered.com/app/{entry["appid"]}\n')

```

Comment:
I think the game name that comes from Steam is already formatted for example 
"Deep Rock Galactic" or "HITMAN 3" 

Unless entry['name'] doesn't contain that value?

Reply:
It does but some are all caps and some aren't so I thought it looked neater if we ran it through .title() for cohesion, and functionally we run user_input through .lower() so it doesn't mess with how a user would write their game name

Reply:
Fair enough, but one is for matching, ease of use, and the other is a design choice taken by the publisher of the game, and how users will see it in other contexts.

Reply:
True, I suppose keeping in line with publisher decisions trumps aesthetics. Shall revert.